twinify/napsu_mq/tests/mst_test.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 twinify Developers and their Assignees
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from twinify.napsu_mq.mst import *
+
+
+class MSTTest(unittest.TestCase):
```

Comment:
yea, I guess we can delete this one

Reply:
MST should have some tests if we include it in the public API.

Reply:
True. I was thinking because it seems mainly taken from a different source, it would already be (somewhat) tested. But that might be a strong assumption and of course we should have (at least) the same tests to ensure we don't break anything with potential future changes..